dali/pipeline/executor/executor.cc

```diff
@@ -579,11 +579,27 @@ void Executor::SetupDataForGraph(WorkspaceBlob *wsb) {
 void Executor::PresizeData(WorkspaceBlob *wsb) {
   TimeRange tr("[Executor] PresizeData");
   DeviceGuard g(device_id_);
+
+  auto mem_hints = [&](OpNode &node) {
+    std::vector<int> hints;
+    int noutputs = node.spec.NumOutput();
+    GetSingleOrRepeatedArg(node.spec, &hints, "bytes_per_sample_hint", noutputs);
+    for (auto &h : hints) {
+      if (h == 0)
+        h = this->bytes_per_sample_hint_;
+    }
+    hints.resize(noutputs, bytes_per_sample_hint_);
```

Comment:
Won't that replace what GetSingleOrRepeatedArg did?

Reply:
It won't be, GetSingleOrRepeatedArg will return noutputs always.

Reply:
Oh, I see. It's far worse than I though - it actually _throws_ an exception when the size is other than 1 or repeat_count...